e2e/tool-panel.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { test, expect } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+test.describe('Tool Panel', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(
+      '/learn/javascript-algorithms-and-data-structures/basic-javascript/increment-a-number-with-javascript'
+    );
+  });
+  test('should display "//running tests" in console after clicking "Run the Tests (Ctrl+Enter)" button', async ({
+    page
+  }) => {
+    await page.getByTestId('run-the-tests-btn').click();
```

Comment:
I suggest we use the getByRole method here as it is more appropriate and better for accessibility.

Reply:
Playwright is unable to locate a button with text containing round brackets '()' inside, so I used `translations.buttons['run-test’]` and removed the round brackets along with their content, leaving only: 'Run the Tests'.